dash-pipeline/bmv2/dash_metadata.p4

```diff
@@ -28,11 +28,18 @@ struct conntrack_data_t {
     bool allow_out;
 }
 
+enum bit<16> dash_tunnel_dscp_mode_t {
+    INVALID = 0,
+    PRESERVE_MODEL = 1,
+    PIPE_MODEL = 2
```

Comment:
somehow I feel `PIPE_MODE` might not be that intuitive.... but will `EXPLICIT_MODE` be a better name? not sure if it is more clear... but no strong opinion.

Reply:
Please ignore my comment above. What you are doing should be right.